src/lib.rs

```diff
@@ -143,6 +143,90 @@ pub struct Control<'a> {
     ioctl_state: &'a Cell<IoctlState>,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PowerManagementMode {
+    /// Custom, officially unsupported mode. Use at your own risk.
+    /// All power-saving features set to their max at only a marginal decrease in power consumption
+    /// as oppposed to `Aggressive`.
+    SuperSave,
+
+    /// Aggressive power saving mode.
+    Aggressive,
+
+    /// The default mode.
+    PowerSave,
+
+    /// Performance is prefered over power consumption but still some power is conserved as opposed to
+    /// `None`.
```

Comment:
Should this say "over power save"? Apologies for commenting on a closed PR, but just noticed when looking at the changes merged :pray: